packages/hoprd/package.json

```diff
@@ -22,9 +22,9 @@
   },
   "scripts": {
     "clean:wasm": "make -C crates clean",
-    "build:wasm": "make -C crates all && make -C crates install",
+    "build:wasm": "make -s -C crates all && make -s -C crates install",
     "test:wasm": "make -C crates test",
-    "clean": "yarn clean:wasm && rimraf ./lib ./tsconfig.tsbuildinfo ./tsconfig.spec.tsbuildinfo",
+    "clean": "yarn clean:wasm && rm -Rf ./lib ./tsconfig.tsbuildinfo ./build ./rest-api-v2-full-spec.json",
```

Comment:
Wonderful, time to get rid of `rimraf`

Reply:
Going to remove the last occurences as well